### PR TITLE
show warning on trs-workflow import for anonymous users

### PR DESCRIPTION
**Pull request by OlegZharkov**
Since we want to deploy auto-import [buttons](https://gist.github.com/bgruening/a891880ba98534f11ceb42be73d90646). We need some sort of warning for unregistered users

### CODE REVIEW

File: client/src/components/Workflow/TrsImport.vue [hunk 1]
```diff
@@ -23,6 +23,9 @@
                 <trs-tool :trs-tool="trsTool" v-if="trsTool" @onImport="importVersion(trsTool.id, $event)" />
             </div>
         </b-card>
+        <b-alert v-else show variant="danger" class="text-center my-2"
+            >Only registered users can import workflows!</b-alert
```
Comment (mvdbeek) [inline client/src/components/Workflow/TrsImport.vue]: Can you rephrase this to `Anonymous user cannot import workflows, please log in.` or something similar ? That seems like it would be easier to understand for users.

File: client/src/components/Workflow/TrsImport.vue [hunk 2]
```diff
@@ -72,6 +76,9 @@ export default {
         hasErrorMessage() {
             return this.errorMessage != null;
         },
+        isRegistered() {
```
Comment (mvdbeek) [inline client/src/components/Workflow/TrsImport.vue]: I think it might be better to have this be `isAnonymous`, since not all users need to be registered to be able to import workflows (e.g. with auto-registration or other fancy/weird setups). I think in practice that won't make a difference, but it more closely reflects what you're checking here.

File: client/src/components/Workflow/TrsImport.vue [hunk 3]
```diff
@@ -72,6 +76,9 @@ export default {
         hasErrorMessage() {
             return this.errorMessage != null;
         },
+        isAnonymous() {
+            return !getGalaxyInstance().user.isAnonymous();
```
Comment (mvdbeek) [inline client/src/components/Workflow/TrsImport.vue]:
```suggestion
            return getGalaxyInstance().user.isAnonymous();
```

File: client/src/components/Workflow/TrsImport.vue [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <template>
     <div>
-        <b-card title="GA4GH Tool Registry Server (TRS) Workflow Import">
+        <b-card v-if="isAnonymous" title="GA4GH Tool Registry Server (TRS) Workflow Import">
```
Comment (mvdbeek) [inline client/src/components/Workflow/TrsImport.vue]:
```suggestion
        <b-card v-if="!isAnonymous" title="GA4GH Tool Registry Server (TRS) Workflow Import">
```

Comment (OlegZharkov) [reply client/src/components/Workflow/TrsImport.vue]: rights, sorry. Lost freshness of vision

**Comment by jmchilton** [comment]
Awesome - thanks! Feel free to backport this 20.09 if that helps with the deployment.

**Comment by OlegZharkov** [comment]
> Awesome - thanks! Feel free to backport this 20.09 if that helps with the deployment.

thanks John, will do
